### ISSUE TITLE
Client seems to stop receiving most updates after some time in operation

### ISSUE THREAD

**Issue by sachssem** [tdlib/td#2930]
Hey, 

I've been using the Node.js bindings implementation of tdlib (https://github.com/Bannerets/tdl) for 3-4 months now without any issues (hosted on a cloud server).
But at the start of this month I experienced some issues. At first my app stopped receiving updates at all, after updating the bindings library and the prebuilt-tdlib to the then newly released versions it worked again. But soon after it started to behave strangely in the cloud:

 - When rebooting the container everything works fine for some minutes
 - Then somehow most groups and channels stop receiving updates, only a few messages get delivered in the course of a day (although it should be multiple hundreds)
 - The weird thing is that all of my own messages which I send to myself (or via another account) are received instantly. No matter if they are sent as direct message, in a group or a channel

I started to disable/remove all of my "advanced" features (everything except the 'on update' handler and one 'getMessage' call to be exact), because I've read about other issues where the receive function was blocked by some never ending API call, but it didn't solve the problem.

Yesterday I increased the verbosity level to 3, but I can't find any obvious issues in the logs as tdlib seems to be fetching updates all day long:
[ 3][t 0][1718150628.588677167][Client.cpp:280][&td_requests]   Begin to wait for updates with timeout 10.000000
[ 3][t 0][1718150638.589522123][Client.cpp:293][&td_requests]   End to wait for updates, returning object 0 0x0

My tdl and prebuilt-tdlib versions are up to date:
    "prebuilt-tdlib": "0.1008030.0",
    "tdl": "8.0.0",
    
In another issue there was a similar problem when someone received updates with a huge delay, compared with TG web client or mobile app - and the solution was to update tdlib. But in my case the prebuilt-tdlib version is up to date. Could compiling tdlib myself resolve the problem (using the latest  tdlib version and not the latest prebuilt one)? 

I really don't know what else could be causing this behaviour as it worked without any issues for multiple months and now this weird behaviour occurs, which is hard to debug as I can't even reproduce it locally...

Any ideas on why this happens or how to continue debugging this? Thanks in advance!

**Comment by levlam** [comment]
All apps receive updates in supergroups and channels in the same way, so there can be no app-specific issues.

> a few messages get delivered in the course of a day

It is not possible to receive messages in a channel out of order. If some messages were skipped, then the app will receive `updateChatLastMessage` with `last_message == null` before `updateNewMessage`, but this is unlikely to happen while the app is online.
